### ISSUE TITLE
How can I sync the balance an transactions for a wallet?

### ISSUE THREAD

**Issue by cekoivisto** [sublimator/ripple-lib-java#19]
I have a few questions. The first is why doesn't the Account class have a member that is an ArrayList<Transaction> or something similar that holds all the transactions for that account? From what I saw in the code, there is no way to actually store all the transactions for a wallet. Assuming there isn't a way to store these, is there at least a method for downloading them? I would also like to know if there is a callback for when the account balance is set when you call client.account(...). I see in the code that by calling client.account(...), it automatically requests the account root, but how are we supposed to know when we got a response?

**Comment by sublimator** [comment]
Never got around to it :) 

As stated in the README, the client is pretty crappy, and unless you know what you are doing (read: can modify to fix bugs etc), I wouldn't use it ...

**Comment by cekoivisto** [comment]
Alright, how reliable do you think the core library is? I was planning on using it to generate wallets and create transaction objects from JSON.

**Comment by sublimator** [comment]
Yeah, you're probably OK with that. I have a fair degree of confidence in that, and also, rippled will validate the transactions. Wallet generation is also likely fine.
